jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -408,4 +411,63 @@
               ./hack/jenkins/e2e.sh
             fi
     jobs:
-        - 'kubernetes-pull-{suffix}'
+        - '{git-project}-pull-{suffix}'
+
+- project:
+    name: node-pull
+    git-project:
+        - 'cadvisor':
+            repo-name: 'google/cadvisor'
+            git-basedir: 'github.com/google/cadvisor'
+            suffix: 'build-test-e2e'
+            owner: 'stclair@google.com'
+            max-total: 1
+            trigger-phrase: 'test'
+            status-context: 'Jenkins GCE e2e'
+            cmd: |
```

Comment:
(no action required) We should move this all to a script. Just noting it here...